bower.json

```diff
@@ -29,6 +29,8 @@
   "devDependencies": {
     "angular-mocks": "1.3.16",
     "angular-resource": "1.3.16",
-    "angular-ui-ace": "0.2.3"
+    "angular-ui-ace": "0.2.3",
+    "polymer": "Polymer/polymer#^1.2.0",
+    "paper-input": "PolymerElements/paper-input#^1.0.0"
```

Comment:
Yes, it is, see [here](https://www.polymer-project.org/1.0/docs/start/getting-the-code.html) for comparison.